Resolutions/XenobladeX_Resolution/rules.txt

```diff
@@ -3,169 +3,230 @@ titleIds = 0005000010116100,00050000101C4C00,00050000101C4D00
 name = Resolution
 path = "Xenoblade Chronicles X/Graphics/Resolution"
 description = Changes the resolution of the game. Made by getdls.
-version = 4
+version = 5
 
-
-[Preset]
-name = 1280x720 (Default)
+[Default]
 $width = 1280
 $height = 720
 $gameWidth = 1280
 $gameHeight = 720
-$internalRes = 1.0
-$dither = 0.01
-$scaleShader = 1.0
-$scaleBlur = 0.05
+$internalScale = 1.0
+$shadowScale = 1.0
+$skellViewScale = 1.0
 
-// Quality
 
+# SCREEN RESOLUTION
 [Preset]
-name = 1600x900
-$width = 1600
-$height = 900
-$gameWidth = 1280
-$gameHeight = 720
-$internalRes = 1.0
-$dither = 0.1
-$scaleShader = (720.0/900.0) # factor for any shader that can't handle fractional scaling
-$scaleBlur = 0.5
+name = 720p (Native)
```

Comment:
What's with the changes to the ordering and naming of the resolution presets?

Reply:
Name and order returned to what had existed prior.